modin/pandas/test/dataframe/test_indexing.py

```diff
@@ -2047,6 +2058,44 @@ def _make_copy(df1, df2):
         eval_setitem(modin_df, pandas_df, df_value[["value_col1"]], key)
 
 
+def test_setitem_2d_insertion():
+    md_df, pd_df = create_test_dfs(test_data["int_data"])
```

Comment:
I would prefer we use `modin_df` instead of `md_df` and `pandas_df` instead of `pd_df`.

Reply:
done, renamed all of the instances of `md/pd_df` to `modin/pandas_df`